ui/packages/catalog-ui-search/src/main/webapp/component/query-basic/query-basic.view.js

```diff
@@ -60,6 +62,12 @@ define([
     return nested
   }
 
+  function getMatchTypeAttribute() {
+    return metacardDefinitions.metacardTypes[properties.basicSearchMatchType]
+      ? properties.basicSearchMatchType
+      : 'datatype'
```

Comment:
Do we actually want to be defaulting here if the property is not set? I would think setting the default value in `org.codice.ddf.catalog.ui.config` would be enough.

Reply:
This is more for the case of an invalid metacard attribute name being entered in the config. Choices were to either exclude from the search or fallback to `datatype`. me and @bdeining decided on the latter